.github/workflows/check-projects-comment.yml

```diff
@@ -0,0 +1,63 @@
+name: check-projects-comment
+
+on:
+  workflow_run:
+    workflows: ['check-projects']
+    types:
+      - completed
+
+jobs:
+  upload:
+    runs-on: ubuntu-latest
+    if: >
+      ${{ github.event.workflow_run.event == 'pull_request' &&
+      github.event.workflow_run.conclusion == 'success' }}
+    steps:
+      - name: 'Download artifact'
+        uses: actions/github-script@v3
+        with:
+          script: |
+            const artifacts = await github.actions.listWorkflowRunArtifacts({
+               owner: context.repo.owner,
+               repo: context.repo.repo,
+               run_id: ${{ github.event.workflow_run.id }},
+            })
+            const matchArtifact = artifacts.data.artifacts.filter((artifact) => {
+              return artifact.name == 'pr'
+            })[0]
+            const download = await github.actions.downloadArtifact({
+               owner: context.repo.owner,
+               repo: context.repo.repo,
+               artifact_id: matchArtifact.id,
+               archive_format: 'zip',
+            })
+            const fs = require('fs')
+            fs.writeFileSync('${{github.workspace}}/pr.zip', Buffer.from(download.data))
```

Comment:
Better to use GitHub official CLI to shorten your code:
https://cli.github.com/manual/gh_run_download